packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -162,12 +207,39 @@ function VerificationEducationScreen({ route, navigation }: Props) {
           {t('verificationEducation.header')}
         </Text>
         <Text style={styles.body}>{bodyText}</Text>
+        <PhoneNumberInput
+          label={t('nuxNamePin1:phoneNumber')}
+          style={styles.phoneNumber}
+          country={country}
+          nationalPhoneNumber={getE164DisplayNumber(e164PhoneNumber || '')}
+          editable={false}
+        />
         {firstButton}
         <View style={styles.spacer} />
         <TextButton style={styles.doINeedToConfirmButton} onPress={onPressLearnMore}>
           {t('verificationEducation.doINeedToConfirm')}
         </TextButton>
       </ScrollView>
+      <Modal isVisible={isCaptchaVisible} style={styles.recaptchaModal}>
```

Comment:
i mentioned this to cal, but we should just be mindful of modals from a perf perspective - can lead to overdraw https://developer.android.com/topic/performance/rendering/overdraw